src/main/java/org/alliancegenome/curation_api/jobs/JobScheduler.java

```diff
@@ -188,50 +183,7 @@ public void pendingFileJobs(@Observes PendingBulkLoadFileJobEvent event) {
 
 	@Scheduled(cron = "0 0 0 ? * SAT")
 	public void cleanUpFileExceptions() {
-		SearchResponse<BulkLoadGroup> groups = groupDAO.findAll();
-
-		Comparator<BulkLoadFile> c = new Comparator<BulkLoadFile>() {
-			@Override
-			public int compare(BulkLoadFile blf1, BulkLoadFile blf2) {
-				if (blf1.getDateLastLoaded() == null && blf2.getDateLastLoaded() == null) {
-					return 0;
-				}
-				if (blf1.getDateLastLoaded() == null) {
-					return 1;
-				}
-				if (blf2.getDateLastLoaded() == null) {
-					return -1;
-				}
-				return blf2.getDateLastLoaded().compareTo(blf1.getDateLastLoaded());
-			}
-		};
-
-		int historyExceptionsToKeep = 3;
-
-		ProcessDisplayHelper ph = new ProcessDisplayHelper();
-		ph.startProcess("Bulk Load File History Exception Cleanup");
-
-		for (BulkLoadGroup g : groups.getResults()) {
-			Log.debug("Group Clean Up: " + g.getName());
-			if (g.getLoads().size() > 0) {
-				for (BulkLoad b : g.getLoads()) {
-					Log.debug("Load Clean Up: " + b.getName());
-					Collections.sort(b.getLoadFiles(), c);
-					if (b.getLoadFiles().size() > historyExceptionsToKeep) {
-						for (int i = historyExceptionsToKeep; i < b.getLoadFiles().size(); i++) {
-							Log.debug("File Clean Up: " + b.getLoadFiles().get(i).getDateLastLoaded());
-							for (BulkLoadFileHistory h : b.getLoadFiles().get(i).getHistory()) {
-								for (BulkLoadFileException e : h.getExceptions()) {
-									bulkLoadFileExceptionDAO.remove(e.getId());
-									ph.progressProcess();
-								}
-							}
-						}
-					}
-				}
-			}
-		}
-		ph.finishProcess();
+		bulkLoadFileExceptionDAO.cleanUpTwoWeekOldExceptions();							
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace
<!-- __reviewdog__:ChBkZTIwODU5YjZiN2RiMTVjEgpjaGVja3N0eWxl -->